src/IssueCreator/Dialogs/ManageIssue.cs

```diff
@@ -31,52 +26,40 @@ public ManageIssue(IssueManager issueManager, Settings settings, FileLogger logg
             _issueManager = issueManager;
             _logger = logger;
             _settings = settings;
-
-            cboEpics.Items.AddRange(epicList);
+            if(epicList != null)

```

Comment:
Can this be null?

Reply:
Yes, when it is invoked from `IssueTemplate` since I refactored this case to be the base form for the new `EpicAssociation` and `IssueTemplate` forms.